data/json/snippets.json

```diff
@@ -679,7 +688,8 @@
 "UFO CRASH: TRUTH OR FICTION?  One civilian account claims a shining disk appeared in his field.  \"It kind of glowed, and I could see a strange gray world through it.  Then a shambling hazy purple mushroom came through, man.\"", 
 "TIME TRAVEL: TRUTH OR FICTION?  Leaked federal documents describe the formation of a new federal agency XEDRA, to oversee \"4th Axis technology\" already in use.  Expert opinion suggests time-travel or parallel worlds.", 
 " ...What do you know about surviving in the Wilderness?  If you can't make a snare you don't know TRAP!  Hunt down a copy of TRAPPERS' LIFE and learn about wildlife!...  And how to kill it.  Classic BEAR TRAP returns in this issue!", 
-"HUNTING GOODS!  Food prices getting you down?  Why not get a crossbow or compound bow and Hunt Your Own!?  Our arrows and bolts are completely reusable, so why not hunt animals like Mother Nature intended?"] 
+"HUNTING GOODS!  Food prices getting you down?  Why not get a crossbow or compound bow and Hunt Your Own!?  Our arrows and bolts are completely reusable, so why not hunt animals like Mother Nature intended?",
+"VAULT PROGRAM REVEALED:  XEDRA promises guaranteed admission to secret underground bunker, codenamed VAULT program, to citizens who volunteer to join XEDRA.  The location of the bunker is kept secret to protect it from enemy forces."] 
```

Comment:
Most likely existing federal/XEDRA/corporate employees, I'd imagine.  No need to have it publicly advertised.  (So tabloids, and survivor notes are OK; formal articles not as much.)

Kinda like folks working at Deus Ex's Area 51.  Sure, it's the place from which MJ12 controls the internet, but we don't *tell* people that, we just have 'em working at a highly secure lab facility that happens to do work on "alien" crap.